src/index.js

```diff
@@ -20,7 +20,7 @@ function buildBundles (configs) {
   const promises = configs.map(buildBundle)
   return Promise.all(promises)
     .then((bundles) => {
-      console.log('built %d bundles', configs.length)
+      console.log('[%s] built %d bundles', new Date().toTimeString().substr(0, 8), configs.length)
```

Comment:
Omitting `substr`  would result a format like this: `00:00:00 GMT+0000 (CET)` which I find kind of verbose. What we want is just the time, timezone is not related as it will be always the user's one.